src/parameterizations/radiation/radiation.jl

```diff
@@ -63,7 +63,8 @@ function radiation_model_cache(
         latitude = RRTMGPI.field2array(zero(bottom_coords.z)) # flat space is on Equator
     end
     local radiation_model
-    orbital_data = Insolation.OrbitalData()
+    data_dir = joinpath(pkgdir(CAP), "artifacts")
```

Comment:
this was required as part of the update or is just an additional enhancement?

Reply:
This was required, related issue: https://github.com/CliMA/ArtifactWrappers.jl/issues/12